app/models/searchgov_url.rb

```diff
@@ -61,7 +61,11 @@ def indexed?
 
   def self.fetch_new(delay: 10)
     while unfetched.any?
-      unfetched.first.fetch
+      begin
+        unfetched.first.fetch
+      rescue => error
+        Rails.logger.error "[SearchgovUrl] Unable to index #{url} into searchgov:\n#{error}\n#{error_line}".red
```

Comment:
Oy - that was some bad copy pasta. I'm actually removing the `error_line` from this log line, because it's pretty much always raised by a save error (which is a separate bug to be fixed).